problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -8,23 +8,22 @@ function MentoringWithStepsBlock(runtime, element) {
 
     var children = runtime.children(element);
     var steps = [];
-    var reviewStep;
+
     for (var i = 0; i < children.length; i++) {
         var child = children[i];
         var blockType = $(child.element).data('block-type');
         if (blockType === 'sb-step') {
             steps.push(child);
-        } else if (blockType === 'sb-review-step') {
-            reviewStep = child;
         }
     }
 
     var activeStep = $('.mentoring', element).data('active-step');
-    var reviewTipsTemplate = _.template($('#xblock-review-tips-template').html()); // Tips about specific questions the user got wrong
     var attemptsTemplate = _.template($('#xblock-attempts-template').html());
     var message = $('.sb-step-message', element);
-    var checkmark, submitDOM, nextDOM, reviewDOM, tryAgainDOM,
-        gradeDOM, attemptsDOM, reviewTipsDOM, reviewLinkDOM, submitXHR;
+    var checkmark, submitDOM, nextDOM, reviewButtonDOM, tryAgainDOM,
+        gradeDOM, attemptsDOM, reviewLinkDOM, submitXHR;
+    var reviewStepDOM = $("[data-block-type=sb-review-step]", element);
```

Comment:
It's probably more efficient to restrict this a bit further, like `"div.xblock[data-block-type=sb-review-step]"`.  Otherwise jQuery will have to iterate through all DOM objects and retrieve the `data-block-type` attribute.  (Not sure whether what I suggested would be correct in all cases.  Also not sure whether performance matters here.)